packages/web/src/components/PageBanner.tsx

```diff
@@ -22,12 +25,14 @@ const PageBanner = ({ children, color, backButtonAction }: Props) => {
 
   return (
     <Box bgcolor={currentBannerColor || colors.black} mb={4} py={2}>
-      <Grid container wrap="nowrap" alignItems="center" spacing={1}>
-        <Grid item>
+      <List>
+        <ListItemIcon>
           <BackButton backButtonAction={backButtonAction} />
-        </Grid>
-        <Grid item>{children}</Grid>
-      </Grid>
+        </ListItemIcon>
+        <ListItem>
+          <Grid item>{children}</Grid>
+        </ListItem>
+      </List>
```

Comment:
Few requested updates here:

1. The `ListItemIcon` should be a child of the `<ListItem/>` based on those MUI examples
2. Since we're removing the `<Grid container/>` from this component we don't need the `<Grid item>` here either. We could replace that with `<ListItemText/>` instead. This will get us pretty close the layout we want. After removing this last `<Grid/>` I think we're good to remove that import from the top of this component as well.
3. Finally, we'll probably want to apply the [disablePadding prop](https://v4.mui.com/api/list/#props) to the `<List/>` so we have spacing close to what was there before.

Reply:
👋  hey,
I sent you a slack message. let me know what you think. 🤔

Reply:
hi Jacob,
I just made a new commit with update based on the new notes. let me know how it looks on your end.
thank you. 😊

Reply:
Awesome, looks good so far 🎉 ! I'm going resolve this and comment on anything new (if needed) separately.